kubeadm-ci/Dockerfile

```diff
@@ -2,26 +2,28 @@ FROM ubuntu
 MAINTAINER bjozsa@att.com
 
 ENV container docker
-RUN apt-get -y update
 
-RUN apt-get update -qq && apt-get install -qqy \
-    apt-transport-https \
-    ca-certificates \
-    curl \
-    lxc \
-    vim \
-    iptables
+RUN set -x \
```

Comment:
FYI: I definitely see where you're going here, but we purposely quieted the output of these commands for CI...to limit the noise seen from Travis to just what's needed.